web/apps/labelstudio/src/pages/CreateProject/Config/Preview.jsx

```diff
@@ -126,18 +122,15 @@ export const Preview = ({ config, data, error, loading, project }) => {
       });
 
       store.annotationStore.selectAnnotation(c.id);
-      console.log("LSF task updated");
+      console.log("LSF updated");
```

Comment:
this is a good opportunity to remove this to reduce the console flood - alternatively maybe we can do something like 
```suggestion
      if(window.APP_SETTINGS.debug) {
            console.log("LSF updated");
      }
```

Reply:
good idea, but maybe not to do it via APP_SETTINGS, which is coming from a backend. we should be able to debug with clients so we have to be able to enable debug in runtime. we still can do `window.APP_SETTINGS.debug = true` though